test/compilable/test19657.sh

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env bash
+
+TEST_DIR=${OUTPUT_BASE}
```

Comment:
TEST_DIR is an _existing_ exported variable.

https://github.com/dlang/dmd/blob/4edae68767fcfd9b12ea4b638cb68e181dc0de49/test/tools/exported_vars.sh#L3

While possible, probably not a good idea to overwrite it.